### PR TITLE
Table of Contents rendered at the bottom fixed and rendered at the top of the page

### PR DESCRIPTION
**Pull request by abhinavsri360**
A meaningful title for PR, not like: made changes to xyz.md

Fixes #
I have fixed the position of Table of Contents in the following pages namely:
Registers, MSI, Flipflops, and Combinational. In all these pages Table of Contents was rendered at the bottom of the page, I have fixed that and made Table of contents render at the top of the page.
Also, In all the sub-pages of these pages, some of them were not having Table of Contents in them, I have fixed that as well.
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/Latches
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/register
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/combinational
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/msi
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/registers/pp
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/Combinational/half_sub
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop/t_flipflop
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop/sr_flipflop
https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/registers/sp
Ref #

#### Changes done:
Table of Contents rendered at the bottom, fixed in the pages mentioned above.

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add a link from the checks tab after checks complete)
- [x] Tried Squashing the commits into one
All changes are present in the commit: 7eb4576

### CODE REVIEW

**Comment by abhinavsri360** [comment]
@YashKumarVerma I have made the necessary changes, now you can review this PR

**Comment by nitin10s** [comment]
It's best you first open issues and confirm it from one of the mentors prior to working on that issue.

**Comment by abhinavsri360** [comment]
Should I open one now?

File: docs/Combinational/combinational.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ permalink: /docs/combinational
...
 {: .no_toc }
 
+## Table of Contents
+
+- [Half Adder](https://learn.circuitverse.org/docs/Combinational/half_adder.html)
+- [Full Adder](https://learn.circuitverse.org/docs/Combinational/full_adder.html)
+- [Half Subtractor](https://learn.circuitverse.org/docs/Combinational/half_sub.html)
+- [Full Subtractor](https://learn.circuitverse.org/docs/Combinational/full_sub.html)
+- [N-bit Parallel Adder and Subtractor](https://learn.circuitverse.org/docs/Combinational/N-Bit%20Parallel%20Adder%20&%20Subtractor.html)
+
```
Comment (YashKumarVerma) [inline docs/Combinational/combinational.md:18]: I do not think ToC's have been implemented like this on other pages.

Comment (abhinavsri360) [reply docs/Combinational/combinational.md]: You were right, I have fixed the numbering now. Please find the deployment link below.

**Comment by abhinavsri360** [comment]
Deployment link: https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/combinational
Please toggle between the pages. The numbering format is resolved now.
Waiting for review @YashKumarVerma

**Comment by abhinavsri360** [comment]
Extremely Sorry for that @YashKumarVerma, I should take care of everything.
I have fixed that, please let me know if any more changes are required.
Here is the deployment link: https://deploy-preview-332--tender-ardinghelli-ebb79f.netlify.app/docs/latches

**Comment by abhinavsri360** [comment]
Title fixed
Latest Deployment Link: https://learn.circuitverse.org/docs/MSI/decoder.html

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions.

**Comment by abhinavsri360** [comment]
Thank you so much :smile:

**Pull request merged**